frontend/src/index.css

```diff
@@ -1,3 +1,76 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+@layer base {
+  :root {
```

Comment:
Thanks for making it ideal for both the themes, couple of doubts.
1. Do we actually need two for skeletons ? Wouldn't grey work on both the themes
2. If we are using both the themes how are we planning to use this using tailwind-classes does it directly take from the theme, or do we need to set it somewhere.

If you could share the Gif of both the light and dark theme, would be really helpful for us to take a call about which one would best suit us, and take a effective call if we need for both the themes explicitly

Reply:
Thanks, for taking time and reviewing it.
I have not made something for both themes these are changes while setup. 
I have created shadcn's UI setup as per steps in doc
https://ui.shadcn.com/docs/installation/vite
After the installation following files are changed 
1. vite.config.ts
2. tailwind.config.js
3. src/index.css
If the some setup will be used in future but for now if we want minimal setup we can do that :). Please give some tips how to keep it simple.

Reply:
Oh Okay, But we would be not requiring all the said changes actually, because it is setup used to install shadcn with vite, but we already have vite installation with vite and we are trying to install shadcn on top of the exiciting vite, so all the said changes wouldn't be required, so we would have to evaluate which we really need and then add.

Reply:
Yes, I will revert tailwind.config.js and index.css.